Interfaces/MainMenu.xib

```diff
@@ -668,6 +668,12 @@ Gw
                                     <action selector="toggleCursorGuide:" target="-1" id="Vxk-ww-Ugb"/>
                                 </connections>
                             </menuItem>
+                            <menuItem title="Show Vertical Cursor Guide" identifier="Show Vertical Cursor Guide" id="pPk-Wb-7b8">
```

Comment:
I was a little confused at first because I thought you could use the vertical guide without the horizontal, but it looks like you always get the horizontal guide but you can also enable the vertical guide in addition.

I think there should be two menu items:
1. Horizontal cursor guide
2. Vertical cursor guide

If neither is on, then there is no guide.

Reply:
I agree, they should be orthogonal concerns.  It was a logical mistake that led to their one-way coupling.  The latest commit to this branch has resolved this.